src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -83,7 +83,7 @@
                 </div>
               </div>
               <div class="col-md-6">
-                <button title="Enroll" id="button_enroll" name="button_enroll"
+                <button type="button" title="Enroll" id="button_enroll" name="button_enroll"
```

Comment:
You need the type button when it's already a `button` element meh?

Reply:
The element resides in a form so it will still "submit" if I click it, placing this type button would prevent that.

Reply:
I see. You can also use `event.preventDefault()` to stop the default action.